### ISSUE TITLE
⛔ 🛟 The new accordion widget is not SEO friendly

### ISSUE THREAD

**Issue by marcelostrappini** [elementor/elementor#33598]
### Description

When using the accordion widget, the text doesn't display correctly when JavaScript and CSS are disabled, demonstrating that it's not SEO-friendly.

As a result, only the titles are visible alongside the paragraph or content of the active item.

### Steps to reproduce

Create a page and use the accordion regularly.

### Expected behavior

Disabling JavaScript and CSS should display:

**First element title**
First element paragraph
**Second element title**
Second element paragraph

### Elementor System Info

```txt

== Server Environment ==
	Operating System: Linux
	Software: nginx/1.18.0
	MySQL version: (Ubuntu) v8.0.44
	PHP Version: 8.4.14
	PHP Memory Limit: 768M
	PHP Max Input Vars: 5000
	PHP Max Post Size: 256M
	GD Installed: Yes
	ZIP Installed: Yes
	Write Permissions: All right
	Elementor Library: Connected

== WordPress Environment ==
	Version: 6.8.3
	Site URL: 
	Home URL: 
	WP Multisite: No
	Max Upload Size: 256 MB
	Memory limit: 40M
	Max Memory limit: 768M
	Permalink Structure: /%postname%/
	Language: es_CL
	Timezone: -3
	Admin Email: 
	Debug Mode: Active

== Theme ==
	Name: Hello Elementor
	Version: 3.4.5
	Author: Elementor Team
	Child Theme: No

== User ==
	Role: administrator
	WP Profile lang: es-CL
	User Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/142.0.0.0 Safari/537.36

== Active Plugins ==
	Classic Editor
		Version: 1.6.7
		Author: WordPress Contributors

	Cloudflare
		Version: 4.12.8
		Author: Cloudflare, Inc.

	Converter for Media
		Version: 6.3.0
		Author: matt plugins

	Disable &quot;BIG image&quot; Threshold
		Version: 1.0
		Author: Jonathan Desrosiers

	Duplicate Page
		Version: 4.5.6
		Author: mndpsingh287

	Elementor
		Version: 3.32.5
		Author: Elementor.com

	Head &amp; Footer Code
		Version: 1.4.5
		Author: Aleksandar Urošević

	Joinchat
		Version: 6.0.8
		Author: Creame

	Meta pixel for WordPress
		Version: 4.1.5
		Author: Facebook

	Perfmatters
		Version: 2.5.1
		Author: forgemedia

	Rank Math SEO
		Version: 1.0.256
		Author: Rank Math SEO

	SVG Support
		Version: 2.5.14
		Author: 

	W3 Total Cache
		Version: 2.8.14
		Author: BoldGrid

	WP Mail SMTP
		Version: 4.6.0
		Author: WP Mail SMTP


== Uso de los elementos ==
	
	footer : 0
		container : 6
		heading : 2
		icon : 4
		image : 1
		nav-menu : 1
	header : 0
		button : 1
		container : 1
		nav-menu : 1
		theme-site-logo : 1
	wp-page : 1
		button : 1
		container : 55
		form : 1
		gallery : 1
		heading : 25
		image : 5
		image-carousel : 1
		nested-accordion : 1
		text-editor : 18


== Ajustes ==
	
	cpt_support: post, page
	disable_color_schemes: yes
	allow_tracking: yes
	unfiltered_files_upload: 1
	font_display: swap
	meta_generator_tag: 1


== Experimentos de Elementor ==
	Fuentes de iconos integradas: Activo por defecto
	Puntos de ruptura personalizados adicionales: Activo por defecto
	Contenedor: Activo por defecto
	Marcado optimizado: Activo por defecto
	Cabecera y pie de página: Inactivo
	Pro Free Trial Popup: Activo por defecto
	Elementos anidados: Activo por defecto
	Panel de páginas: Inactivo por defecto
	Pantalla de inicio de Elementor: Activo por defecto
	Widgets atómicos: Inactivo por defecto
	Clases globales: Inactivo por defecto
	Hacer cumplir las capacidades de las clases globales: Activo por defecto
	Variables: Activo por defecto
	Biblioteca en la nube: Activo
	Editor v4 (página de aceptación): Activo por defecto
	Editor V4: Inactivo por defecto
	Componentes: Inactivo por defecto
	Personalización de importación/exportación: Activo por defecto
	Eventos del editor de Elementor: Activo por defecto


== Registro ==
	
PHP: showing 4 of 4PHP: 2025-09-24 16:54:13 [notice X 37][wp-content/plugins/elementor/includes/base/controls-stack.php::230] hexdec(): Passing null to parameter #1 ($hex_string) of type string is deprecated [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2025-09-29 14:19:47 [warning X 12][wp-content/plugins/elementor/modules/history/revisions-manager.php::363] Undefined array key &quot;editor_post_id&quot; [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2025-09-29 15:50:30 [warning X 3][wp-content/plugins/elementor/includes/managers/image.php::113] Trying to access array offset on false [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2025-10-02 12:08:40 [error X 7][wp-content/plugins/elementor/modules/cloud-library/module.php::190] Uncaught WpOrg\Requests\Exception\Http\Status403: 403 Forbidden in wp-content/plugins/elementor/modules/cloud-library/module.php:190
Stack trace:
#0 wp-content/plugins/elementor/modules/cloud-library/module.php(53): Elementor\Modules\CloudLibrary\Module-&gt;is_screenshot_proxy_mode()
#1 wp-content/plugins/elementor/core/base/module.php(86): Elementor\Modules\CloudLibrary\Module-&gt;__construct()
#2 wp-content/plugins/elementor/core/modules-manager.php(64): Elementor\Core\Base\Module::instance()
#3 wp-content/plugins/elementor/includes/plugin.php(709): Elementor\Core\Modules_Manager-&gt;__construct()
#4 wp-content/plugins/elementor/includes/plugin.php(627): Elementor\Plugin-&gt;init_components()
#5 wp-includes/class-wp-hook.php(324): Elementor\Plugin-&gt;init()
#6 wp-includes/class-wp-hook.php(348): WP_Hook-&gt;apply_filters()
#7 wp-includes/plugin.php(517): WP_Hook-&gt;do_action()
#8 wp-settings.php(727): do_action()
#9 wp-config.php(66): require_once(&#039;...&#039;)
#10 wp-load.php(50): require_once(&#039;...&#039;)
#11 wp-blog-header.php(13): require_once(&#039;...&#039;)
#12 index.php(17): require(&#039;...&#039;)
#13 {main}
  thrown [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]

JS: showing 7 of 7JS: 2025-09-29 17:19:46 [error X 17][/wp-content/plugins/elementor/assets/js/packages/editor-documents/editor-documents.min.js?ver=3.32.2:1:4903] Cannot read properties of null (reading &#039;editor&#039;) 
JS: 2025-09-29 18:55:56 [error X 1][/wp-content/plugins/elementor/assets/js/editor.min.js?ver=3.32.2:3:22587] Cannot read properties of undefined (reading &#039;get&#039;) 
JS: 2025-09-30 02:44:05 [error X 2][/wp-content/plugins/elementor/assets/js/editor.min.js?ver=3.32.3:3:914529] Failed to execute &#039;querySelector&#039; on &#039;Document&#039;: &#039;#&#039; is not a valid selector. 
JS: 2025-09-30 19:31:48 [error X 2][/wp-content/plugins/elementor/assets/js/editor.min.js?ver=3.32.3:3:348359] Cannot read properties of undefined (reading &#039;id&#039;) 
JS: 2025-09-30 19:42:44 [error X 1][/wp-content/plugins/pro-elements/assets/js/gallery.1628df47530ab42dafba.bundle.min.js:2:5799] Cannot read properties of undefined (reading &#039;setSettings&#039;) 
JS: 2025-10-01 20:29:59 [error X 6][/wp-content/plugins/elementor/assets/js/editor.min.js?ver=3.32.3:3:1214320] Cannot read properties of null (reading &#039;id&#039;) 
JS: 2025-10-02 20:09:57 [error X 9][/wp-content/plugins/elementor/assets/js/9f88a0d716a691c2448a.bundle.min.js:2:10410] Cannot read properties of null (reading &#039;dataset&#039;) 

Log: showing 20 of 202025-09-29 22:28:58 [info] elementor::elementor_updater Started 
2025-09-29 22:28:58 [info] Elementor/Upgrades - _on_each_version Start  
2025-09-29 22:28:58 [info] Elementor/Upgrades - _on_each_version Finished 
2025-09-29 22:28:58 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.32.2',
  'to' => '3.32.3',
)]
2025-09-29 22:28:58 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor',
  'from' => '3.32.2',
  'to' => '3.32.3',
)]
2025-09-30 19:14:51 [info] elementor-pro::elementor_pro_updater Started 
2025-09-30 19:14:51 [info] Elementor Pro/Upgrades - _on_each_version Start  
2025-09-30 19:14:51 [info] Elementor Pro/Upgrades - _on_each_version Finished 
2025-09-30 19:14:51 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.31.3',
  'to' => '3.32.1',
)]
2025-09-30 19:14:51 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.31.3',
  'to' => '3.32.1',
)]
2025-10-06 11:22:40 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor',
  'from' => '3.32.3',
  'to' => '3.32.4',
)]
2025-10-06 11:22:40 [info] elementor::elementor_updater Started 
2025-10-06 11:22:40 [info] Elementor/Upgrades - _on_each_version Start  
2025-10-06 11:22:40 [info] Elementor/Upgrades - _on_each_version Finished 
2025-10-06 11:22:40 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.32.3',
  'to' => '3.32.4',
)]
2025-10-21 23:23:26 [info] elementor::elementor_updater Started 
2025-10-21 23:23:26 [info] Elementor/Upgrades - _on_each_version Start  
2025-10-21 23:23:26 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor',
  'from' => '3.32.4',
  'to' => '3.32.5',
)]
2025-10-21 23:23:26 [info] Elementor/Upgrades - _on_each_version Finished 
2025-10-21 23:23:26 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.32.4',
  'to' => '3.32.5',
)]



== Elementor - Compatibility Tag ==
```

### Agreement

- [ ] I confirm I have read and followed all the guidelines and instructions outlined in the Elementor Bug Report form.
- [x] I agree that my issue may be closed without further action if it doesn't meet all the requirements outlined in the Elementor Bug Report form.

**Comment by nicholaszein** [comment]
Hi. Elementor uses native HTML5 `<details>` Elements. By default if they are closed, you'll only see the content of the `<summary>` tag inside them. This means even without CSS and JS, it should behave exactly how it does with CSS and JS. It visually hides the content until you click to reveal it. Search Engines and most importantly Accessibility Tools such as Screen Readers can still read the content inside them.

Please, read the article below to understand how these tags work.
https://developer.mozilla.org/en-US/docs/Web/HTML/Reference/Elements/details

I'm closing this issue as it's not a bug caused by Elementor. It's the intended behavior and why these tags were selected to be used in the first place. In fact the JS is only there to assist in animating and accessibility, but it isn't required at all. The tag works natively the way it was intended to be used. 

Best regards
